app/models/application_qualification.rb

```diff
@@ -60,6 +60,13 @@ class ApplicationQualification < ApplicationRecord
   validates :qualification_type, length: { maximum: MAX_QUALIFICATION_TYPE_LENGTH }, allow_blank: true
   validates :non_uk_qualification_type, length: { maximum: MAX_QUALIFICATION_TYPE_LENGTH }, allow_blank: true
 
+  enum enic_reason: {
+    yes: 'yes',
+    waiting: 'waiting',
+    future: 'future',
+    no: 'no',
+  }, _prefix: :enic_reason
```

Comment:
`enic_reason` feels to me like it should give a reason for enic, this feels more like an `enic_status` or something? I know the migration has already been done and I could be misunderstanding here but what do you think?

Also just to clarify you mentioned `_suffix` in the PR description but it's `_prefix` here, not sure which one you meant to do. I think `_prefix` is nicer here, like you've done.

Reply:
So what will `ApplicationQualification.last.enic_reason_yes?` mean? Does that mean they provided an enic statement or a reason for not providing an enic statement?

Reply:
Maybe a compromise would be `enic_answer` to make it specific that this is the Answer to this question and doesn't necessarily mean that there is an ENIC Reference

Reply:
How about changing `yes` to `statement_provided` and `no` to `statement_not_provided`

I'm not sure what the difference between `waiting` and `future` is 🤔 

Ideally the column would be `enic_statement` then we could have `provided`, `not_provided` etc but that's probably a pain now.

Reply:
enum values:
```ruby
{
  obtained: "", # I have obtained a statement of comparibility
  waiting: "", # I am waiting for a statement of comparibility
  maybe: "", # I might obtain a statement of comparibility (in the future)
  not_needed: "" # I don't need a statement of comparibility
}
```

Neither `enic_reason` or `enic_statement` is great - lets stick with `enic_reason` at the moment for simplicity - but update the enum values away from "yes", "no", etc